users/tests.py

```diff
@@ -1,3 +1,70 @@
-from django.test import TestCase
+import json
+import bcrypt
 
-# Create your tests here.
+from django.test import TestCase, Client
+
+from .models import User, Grade
+from my_settings import password_validation
+
+client = Client()
+
+class UserSignUpTest(TestCase):
+    def setUp(self):
+        clean_password = bcrypt.hashpw("wecode123".encode("utf-8"), salt = bcrypt.gensalt()).decode()
+        Grade.objects.create(name = "일반", accur_rate = 1, id = 6)
+        User.objects.create(
+            email    = "ddalkigum@gmail.com", 
+            name     = "딸기검", 
+            password = clean_password, 
+            address  = "테헤란로 81길"
+            )
+        
+    def tearDown(self):
+        User.objects.all().delete()
+
+    def test_user_sign_up_success(self):
+        clean_password = bcrypt.hashpw("wecode123".encode("utf-8"), salt = bcrypt.gensalt()).decode()
```

Comment:
`password`를 `hashing`해서 전달해주는 이유가 있나요?
```suggestion
        clean_password = bcrypt.hashpw("wecode123".encode("utf-8"), salt = bcrypt.gensalt()).decode()
```

Reply:
저희가 비밀번호를 hashing해서 저장해 주기 때문에 테스트로 해싱한 걸 확인해야 할 것 같아서 저장했습니다